laravel/routes/web.php

```diff
@@ -15,37 +15,26 @@
 
 Route::get('/', 'TopController@index')->name('top');
 
-Route::get('/incoming', 'StockController@incoming')->name('incoming');
+Route::get('/csv.imports', 'StockController@csv_imports')->name('csv_import');
 
-Route::post('/order_imports', 'StockController@order_csv_import')->name('incoming');
+Route::post('/order.imports', 'StockController@order_csv_import')->name('csv_import');
 
-Route::post('/inventory_imports', 'StockController@inventory_csv_import')->name('incoming');
+Route::post('/inventory.imports', 'StockController@inventory_csv_import')->name('csv_import');
 
-Route::get('/orders', 'StockController@order')->name('order');
+Route::get('/orders/index', 'StockController@order')->name('order');
```

Comment:
URLにはindexを含めないほうが良く、メソッド名のほうにindexをつけましょう
(他も同様)

あと本来ならOrderControllerを作り、そちらにindexメソッドを作るほうが
RESTful APIの設計に則っているのでより良い設計になります
```suggestion
Route::get('/orders', 'StockController@order_index')->name('order');
```